lib/mpool/include/mpool/mpool_structs.h

```diff
@@ -52,6 +52,7 @@ struct mpool_cparams {
  */
 struct mpool_rparams {
     struct {
+        bool dio_disable;
```

Comment:
Yikes, this is like 4K per mclass...

Reply:
I guess these buffers could converted to flexible arrays?

Reply:
Maybe not a big deal, I was thinking they were stored in an rparams structure somewhere, but I'm not seeing that...

Reply:
Yes, struct mpool_c/r/dparams are not embedded in any long-lived structs.

At kvdb creation, these paths are serialized into the kvdb.meta file. They are de-serialized from this file at runtime on a need basis.